
async function getTeachers(req, res) {
try {
    
} catch (error) {
    return res.status(500).json({
        ok: false,
        message: "Something goes wrong"
    })
}  
}